charts/fluent-bit/values.yaml

```diff
@@ -140,6 +140,8 @@ extraVolumes: []
 
 extraVolumeMounts: []
 
+extraLuaScripts: {}
```

Comment:
```suggestion
luaScripts: {}
```


nit: Extra would imply that there are some already, afaik fluent bit does not ship with any built in lua scripts today?

Reply:
yeah, there is no any scripts out of the box, renamed